app/components/post_list/more_messages_button/more_messages_button.tsx

```diff
@@ -169,18 +189,22 @@ export default class MoreMessageButton extends React.PureComponent<MoreMessagesB
     show = () => {
         if (!this.buttonVisible && this.state.moreText && !this.props.deepLinkURL && !this.canceled && this.props.unreadCount > 0) {
             this.buttonVisible = true;
-            const toValue = this.indicatorBarVisible ? MAX_INPUT : MAX_INPUT - INDICATOR_BAR_FACTOR;
-            Animated.spring(this.top, {
-                toValue,
-                useNativeDriver: true,
-            }).start();
+            this.animateButton();
         }
     }
 
+    getBarsFactor = () => {
```

Comment:
@jespino if I understand this right, the blue bar that allows you to scroll to the new message line indicator will display below the call indicator. yes ?

Reply:
Yes, bellow the indicator, or bellow the indicator plus the current call floating widget.